cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -128,7 +153,7 @@ public void tryConnect(ConnectionConfig connectionConfig, PrintStream output, bo
   }
 
   public void updateAccessToken(PrintStream output) throws IOException {
-    AccessToken newAccessToken = getNewAccessToken(clientConfig.getConnectionConfig(), output, false);
+    AccessToken newAccessToken = getNewAccessToken(clientConfig, clientConfig.getConnectionConfig(), output, false);
```

Comment:
If you are passing the `clientConfig`, no need to pass `clientConfig.getConnnectionConfig()`.